backends/logging-jul-backend/README.md

```diff
@@ -0,0 +1,14 @@
+## `java.util.logging` (JUL) backend
+
+This module provides the implementation of the logging backend that outputs
```

Comment:
Can we have a shorter directory names under `backends` someday? 

I'm not suggesting to change it in this PR, but `backends/jul` reads better than `backends/logging-jul-backend`.

Reply:
Also, a module ID which reads `spine-logging-backend-jul` also looks more natural than `spine-logging-jul-backend`. The former (prefix) for groups all the backends in alphabetical sorting.

Reply:
Also, it would follow the way we structure our packages in this project.

Reply:
Yes, I agree. These long module names should be shortened.